forge-game/src/main/java/forge/game/card/CardFactoryUtil.java

```diff
@@ -2208,8 +2208,8 @@ public static void addReplacementEffect(final KeywordInterface inst, final CardS
 
             inst.addReplacement(re);
         } else if (keyword.equals("Compleated")) {
-            String sb = "etbCounter:LOYALTY:PhySpent:CheckSVar$ PhySpent | SVarCompare$ LT0:This planeswalker" +
-                    " enters with two fewer loyalty counters for each Phyrexian mana symbol life was paid for";
+            String sb = "etbCounter:LOYALTY:PhySpent:CheckSVar$ PhySpent | SVarCompare$ LT0:Compleated (This planeswalker" +
```

Comment:
get the reminder text from the instance?

Reply:
done